tests/test_utils.py

```diff
@@ -117,6 +121,45 @@ def test_getExpPositionOffset(self):
             self.assertAlmostEqual(result.deltaDec.asDegrees(), deltaDec, 6)
 
 
+class ImageBasedTestCase(lsst.utils.tests.TestCase):
+    """A test case for testing sky position offsets for exposures."""
+
+    @classmethod
+    def setUpClass(cls):
+        try:
+            cls.butler = makeDefaultLatissButler()
+        except FileNotFoundError:
+            raise unittest.SkipTest("Skipping tests that require the LATISS butler repo.")
+
+        cls.dataId = {'day_obs': 20200315, 'seq_num': 120, 'detector': 0}
+        cls.bestEffort = BestEffortIsr()  # should always succeed if makeDefaultLatissButler works
+        cls.exp = cls.bestEffort.getExposure(cls.dataId)
+
+    def test_fluxFromFootprint(self):
+        footPrintSet = detectObjectsInExp(self.exp)
+        allFootprints = footPrintSet.getFootprints()
```

Comment:
This test doesn't need a butler at all, you could construct the image directly. For example, `afw/tests/test_footprint[1,2].py` place objects in an Image and make the footprints in place.